src/wrappedlibc_private.h

```diff
@@ -195,7 +195,7 @@ GO(__dgettext, pFpp)
 GO(difftime, dFuu)
 GO(dirfd, iFp)
 GO(dirname, pFp)
-GO(div, UFii)
+GOM(div, pFpii)
```

Comment:
As div return a struct, I think it will need the GOS macro to put a `RET 4` instead of `RET`.

Reply:
I studied more about how it actually works and you're right. I fixed it in new commit. Although it's strange that it worked with `RET` instead of `RET 4`.

Reply:
Yeah, when I count the number of error I fixed, I would say it's surprisingly resilient to mistakes!